plugins/swf-backend/src/service/router.ts

```diff
@@ -161,13 +161,11 @@ function setupInternalRoutes(
       params: { swfId },
     } = req;
     const swfData = req.body;
-    const svcResponse = await executeWithRetry(() =>
-      fetch(`${kogitoBaseUrl}:${kogitoPort}/${swfId}`, {
-        method: 'POST',
-        body: JSON.stringify(swfData),
-        headers: { 'content-type': 'application/json' },
-      }),
-    );
+    const svcResponse = await fetch(`${kogitoBaseUrl}:${kogitoPort}/${swfId}`, {
```

Comment:
because if there is an error in the workflow execution it stats retying and this is not what we expect in this case, 
let's say there is a wrong parameter when starting the workflow (github repository URL) the workflow should be in error we can't start retrying.. I was getting this situation a lot when designing the workflow with error and compensation